src/sumo.erl

```diff
@@ -326,3 +310,8 @@ normalize_sort_fields({Name, Order}) ->
   [{Name, Order}];
 normalize_sort_fields(SortFields) when is_list(SortFields) ->
   lists:flatmap(fun normalize_sort_fields/1, SortFields).
+
+%% @private
+get_docs() ->
+  {ok, Docs} = application:get_env(sumo_db, docs),
```

Comment:
you can just use a default value here instead of crashing:

```erlang
application:get_env(sumo_db, docs, []).
```

Wouldn't that work even better?